docs/source/admin/capacity-planning.md

```diff
@@ -0,0 +1,246 @@
+
+| node type   | GB RAM / CPU core |
+| ----------- | ----------------- |
+| n2-highmem  | 8                 |
+| n2-standard | 4                 |
+| n2-highcpu  | 1                 |
+
+### Idleness
+
+Jupyter being an interactive tool means people tend to spend a lot more time reading and thinking than actually running resource-intensive code.
+This significantly affects how much _cpu_ resources a typical active user needs,
+but often does not significantly affect the _memory_.
+
+Ways to think about this:
+
+- More idle users means unused CPU.
+  This generally means setting your CPU _limit_ higher than your CPU _request_.
+- What do your users do when they _are_ running code?
+  Is it typically single-threaded local computation in a notebook?
+  If so, there's little reason to set a limit higher than 1 CPU core.
+- Do typical computations take a long time, or just a few seconds?
+  Longer typical computations means it's more likely for users to be trying to use the CPU at the same moment,
+  suggesting a higher _request_.
+- Even with idle users, parallel computation adds up quickly - one user fully loading 4 cores and 3 using almost nothing still averages to more than a full CPU core per user.
+- Long-running intense computations suggest higher requests.
+
+Again, using mybinder.org as an example—we run around 100 users on 8-core nodes,
+and still see fairly _low_ overall CPU usage on each user node.
+The limit here is actually Kubernetes' pods per node, not memory _or_ CPU.
+This is likely a extreme case, as many Binder users come from clicking links on webpages
+without any actual intention of running code.
+
+![mybinder.org node CPU usage is low with 50-150 users sharing just 8 cores](../images/mybinder-load5.png)
+
+## More tips
+
+### Start strict and generous, then measure
+
+A good tip, in general, is to give your users as much resources as you can afford that you think they _might_ use.
+Then, use resource usage metrics like prometheus to analyze what your users _actually_ need,
+and tune accordingly.
+Remember: **Limits affect your user experience and stability. Requests mostly affect your costs**.
+
+For example, a sensible starting point (lacking any other information) might be:
+
+```yaml
+request:
+  cpu: 0.5
+  mem: 2G
+limit:
+  cpu: 1
+  mem: 2G
+```
+
+(more memory if significant computations are likely - machine learning models, data analysis, etc.)
+
+Some actions
+
+- If you see out-of-memory killer events, increase the limit (or talk to your users!)
+- If you see typical memory well below your limit, reduce the request (but not the limit)
+- If _nobody_ uses that much memory, reduce your limit
+- If CPU is your limiting scheduling factor and your CPUs are mostly idle,
+  reduce the cpu request (maybe even to 0!).
+- If CPU usage continues to be low, increase the limit to 2 or 4 to allow bursts of parallel execution.
+
+(measuring)=
+
+### Measuring user resource consumption
```

Comment:
I'd also recommend adding a section on cloud provider cost monitoring and limits services. I think these are often the simplest way to limit your cloud bill (e.g. "Have AWS send me an email if I hit XXX spending trajectory on week 3 of the month")

Reply:
Good call. Added.